core/enumerator/arithmetic_sequence/endless_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../../../spec_helper'
+
+describe "Enumerator::ArithmeticSequence endless" do
+  it "returns an instance of Enumerator::ArithmeticSequence" do
+    (10..).step(1).should be_an_instance_of(Enumerator::ArithmeticSequence)
+    (10..).step(1).should be_an_instance_of(Enumerator::ArithmeticSequence)
+  end
+end
```

Comment:
Same, this is already tested in https://github.com/oracle/truffleruby/blob/baa94b35af7fca1fcf87981009c4f5896de046d6/spec/ruby/core/range/step_spec.rb#L477-L516

Reply:
removed